test/puppet-mode-test.el

```diff
@@ -601,6 +601,82 @@ package { 'bar':
     (should (local-variable-p 'imenu-create-index-function))
     (should (equal imenu-create-index-function #'puppet-imenu-create-index))))
 
+
+;;;; Indentation
+
+(defmacro flycheck-ert-with-temp-buffer (&rest body)
```

Comment:
@tko It's appropriate, but you should rename the functions, from `flycheck-ert-*` to `puppet-test-*` or something similar.  This just isn't Flycheck, after all :smiley: